tests/lib/markdown.js

```diff
@@ -74,3 +74,11 @@ test('Video Plugin - Renders Vimeo', t => {
 
   t.is(output, expected, 'Vimeo Embed');
 });
+
+test('Video Plugin - Renders Ustream', t => {
+  const sample = '@[http://www.ustream.tv/embed/22046713?html5ui]';
```

Comment:
For both the YouTube and Vimeo embeds, we direct users to copy the URL from the from in-browser and use that to embed, instead of finding the embed URL themselves. This means that we're going to want to work off of URLs like `http://www.ustream.tv/recorded/92458394` instead of what's here. 

Now, Ustream appears to be annoyingly inconsistent here. While `http://www.ustream.tv/channel/zn4QqhwEfxr` is the link to the channel in-browser for the video you're using, if you hit the `Copy Link` share, it gives you `http://www.ustream.tv/channel/22046713`. For recorded videos, the browser URL and the Copy Link URL are the same.

This extension should be able to support both channel URLs and recorded URLs. This will likely also help to resolve the different embed URLs as previously mentioned